src/components/calcComponents/Display.js

```diff
@@ -1,16 +1,12 @@
 import PropTypes from 'prop-types';
 import './Display.css';
 
-const Display = (props) => {
-  const { value } = props;
-  return (
-    <div className="display-container">
-      <p className="value">{value}</p>
-    </div>
-  );
-};
+const Display = ({ value }) => (
+  <div className="display-container">
+    <p className="value" id="display-value">{value}</p>
+  </div>
+);
```

Comment:
- [ ] (Optional) It would be nice to have the mathematical symbols _**(+, -, ÷, x, %)**_ show when performing mathematical operations on the display div. 

For example, when you click '6', then click '+', the display div should show '6+'